src/main/resources/db/migration/V1_78__added_loc_code_to_financial_audit_primary_key.sql

```diff
@@ -0,0 +1,51 @@
+ALTER table juror_mod.financial_audit_details_appearances
+    add column loc_code varchar(3) NOT NULL default 'ERR'
+;
+
+update juror_mod.financial_audit_details_appearances fada
+set loc_code = fad.loc_code
+from juror_mod.financial_audit_details fad
+where fad.id = fada.financial_audit_id
+  and fada.loc_code = 'ERR';
+
+
+ALTER table juror_mod.financial_audit_details_appearances
+    add CONSTRAINT financial_audit_details_appearances_app_loc_code_fk
+        FOREIGN KEY (loc_code) REFERENCES juror_mod.court_location (loc_code),
+    add CONSTRAINT financial_audit_details_appearances_id_loc_code_unique UNIQUE (financial_audit_id, loc_code, attendance_date, appearance_version)
```

Comment:
do we need to drop the existing unique/pk constraint to replace it with this one?

Reply:
Removed the unique constraint and added in a new PK constraint